databooks/cli.py

```diff
@@ -45,9 +84,9 @@ def callback(  # noqa: D103
 callback.__doc__ = _DISTRIBUTION_METADATA["Summary"]
 
 
-@app.command()
+@app.command(add_help_option=False)
```

Comment:
Why do we use our own help functionality? This arg seems to suggest that `typer` has something for this by default

Reply:
Yeah it does, but it's not an `eager` command. So if you type something like `databooks fix --help` it'd complain that you didn't pass `paths` (cause it'd try to get the config - which `is_eager` and depends on the paths).
If we don't reimplement it then only `databooks fix . --help` would work...

Reply:
I'm amazed that this is not the default 😅 Thanks for sharing the knowledge